src/extensions/default/CodeFolding/main.js

```diff
@@ -54,7 +45,11 @@ define(function (require, exports, module) {
         EXPAND                  = "codefolding.expand",
         EXPAND_ALL              = "codefolding.expand.all",
         gutterName              = "CodeMirror-foldgutter",
-        COLLAPSE_CUSTOM_REGIONS = "codefolding.collapse.customregions";
+        codeFoldingMenuDivider  = "codefolding.divider",
+        collapseKey             = "Ctrl-Alt-[",
+        expandKey               = "Ctrl-Alt-]",
+        collapseAllKey          = "Alt-1",
+        expandAllKey            = "Shift-Alt-1";
```

Comment:
On master, I can't seem to override any of these shortcuts using keymap.json.  I can't remember if this is some limitation of the user keybindings feature (e.g. can't override shortcuts defined by an extension? -- cc @RaymondLim)...  But to users it will just feel like a bug since they don't know which things are implemented under the hood as a built-in extension.

If it's tricky to fix we should spin this off as a separate bug though.